### DISCUSSION TITLE
Как сделать чтобы по Ентер не входил в архив

### DISCUSSION THREAD

**Discussion by Catty82** [elfmz/far2l#1111] [General]
Некоторые файлы действительно являются архивами внутри, но имеют расширение отличное от архивного. Фар при нажатии Ентер в них входит и отображает их структуру что не требуется, вместо того чтобы по Ентер отработать открытие приложения с файлом. 
Да, открытие приложения делается по ШифтЕнтер. Потому и вопрос.
Пропись File association не помогает.

**Comment by Mischutka** [comment]
Попробуйте так: https://forum.astralinux.ru/threads/2049/post-16727

**Reply by Catty82** [reply]
Спасибо.
